discovery-infra/tests/test_wrong_boot_order.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+from tests.base_test import BaseTest
+
+NODE = 'master-0-2'
+
+
+class TestWrongBootOrder(BaseTest):
+    @pytest.mark.regression
+    def test_wrong_boot_order_one_master(self, api_client, node_controller):
+        # Define new cluster
+        cluster_id = self.create_cluster(api_client=api_client).id
+        # Generate and download cluster ISO
+        self.generate_and_download_image(cluster_id=cluster_id, api_client=api_client)
+        # Change boot order of master-0-0
+        node_controller.set_boot_order_to_node(node_name=NODE, cd_first=True)
```

Comment:
lets move test to use host object. For now u can list all nodes and choose one of them

Reply:
agreed in next PR